tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -3943,4 +3943,229 @@ public function provide_test_is_valid_optimized_checkout_page() {
+		$this->mock_gateway->add_currency_conversion_notice( $order );
+		$output = ob_get_clean();
+
+		$expected_amount     = WC_Stripe_Helper::get_woocommerce_amount_from_stripe_amount( 1500, 'eur' );
+		$stripe_order_amount = WC_Stripe_Helper::get_stripe_amount( $order->get_total(), $order->get_currency() );
+		$expected_rate       = wc_format_decimal( 1500 / $stripe_order_amount, wc_get_price_decimals() );
+
+		$this->assertStringContainsString( '<p class="woocommerce-info" style="margin-top: 1em;">', $output );
+		$this->assertStringContainsString( $expected_amount . ' EUR', $output );
+		$this->assertStringContainsString( $expected_rate . ' EUR', $output );
+		$this->assertStringContainsString( '</p>', $output );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_currency_conversion_notice_reads_from_order_meta_without_api_call is not named in camelCase. (undefined)

(CamelCaseMethodName)